listen.gemspec

```diff
@@ -33,6 +33,9 @@ Gem::Specification.new do |s|
   s.add_dependency 'rb-fsevent', '>= 0.9.3'
   s.add_dependency 'rb-inotify', '>= 0.9.7'
 
+  # Used to show warnings at runtime
+  s.add_dependency 'ruby_dep', '~> 1.1'
```

Comment:
Does this mean you no longer have to install `ruby_dep` before you can bundle install this gem locally?